docs/reference_guide.md

```diff
@@ -1644,6 +1645,118 @@ fd 3 name libselinux.so.1
 ```
 And as you can see in above example it's also possible to access function arguments on `kretfunc` probes.
 
+## 17. `iter`: Iterators Tracing
+
+**WARNING**: this feature is experimental and may be subject to interface changes.
```

Comment:
Adding a 

'Kernel: 5.4`

would be helpful as a quick ref, some calls/builtins already have it

Reply:
sure, will add